src/handlers/register-wallet.ts

```diff
@@ -58,7 +58,7 @@ export async function registerWallet(context: Context, address: string | null) {
 export async function resolveAddress(ensName: string): Promise<string | null> {
   // Gets the Ethereum address associated with an ENS (Ethereum Name Service) name
   // Explicitly set provider to Ethereum mainnet
-  const provider = new ethers.providers.JsonRpcProvider(`https://eth.llamarpc.com`); // mainnet required for ENS
+  const provider = new ethers.JsonRpcProvider(`https://eth.llamarpc.com`); // mainnet required for ENS
```

Comment:
what at about if the RPC turns down?

Reply:
can we use the dynamic RPC logic here?

Reply:
Good point, this PR just changes the call though. Perhaps this can be addressed in an another pull request?

Reply:
Agree, good point. I just fixed the call from ether 5 to 6 so didn't change any logic, would be worth addressing in another PR definitely